### DISCUSSION TITLE
How can I extend postcss config from a common file?

### DISCUSSION THREAD

**Discussion by ronny-lark** [postcss/postcss#1887] [General]
Imagine I have a monorepo with multiple frontend apps using postcss.
Say they all use similar or the same plugins.
Is there a way to define an object in some common place in the repo and have all the apps' `postcss.config.js` extend it?

**Comment by ai** [comment]
You do not need special feature for that. JS configs support `require`/`import`.

**Reply by ronny-lark** [reply]
Oh, brilliant! Sorry I actually didn't realize it would work like that. Thanks!
